src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -297,6 +302,34 @@ std::string TypeName(const FieldDescriptor* field) {
   }
 }
 
+std::string PhpTypeName(const FieldDescriptor* field, bool is_descriptor) {
+  if (field->is_repeated()) {
+    return "array|\\Google\\Protobuf\\Internal\\RepeatedField";
+  }
+  switch (field->type()) {
+    case FieldDescriptor::TYPE_INT32:
+    case FieldDescriptor::TYPE_INT64:
+    case FieldDescriptor::TYPE_UINT32:
+    case FieldDescriptor::TYPE_UINT64:
+    case FieldDescriptor::TYPE_SINT32:
+    case FieldDescriptor::TYPE_SINT64:
+    case FieldDescriptor::TYPE_FIXED32:
+    case FieldDescriptor::TYPE_FIXED64:
+    case FieldDescriptor::TYPE_SFIXED32:
+    case FieldDescriptor::TYPE_SFIXED64:
+    case FieldDescriptor::TYPE_ENUM: return "int";
```

Comment:
int64, uint64 are represented as numeric string on 32-bit machine. How shall we handle this?